config/locales/translations/zendesk_apps_support.yml

```diff
@@ -119,6 +119,11 @@ parts:
       key: "txt.apps.admin.error.app_build.invalid_requirements_types.other"
       title: "App builder job: requirements file contains invalid types error"
       value: "requirements.json contains invalid types: %{invalid_types}"
+  - translation:
+      key: "txt.apps.admin.error.app_build.unsupported_mime_type_detected"
+      title: "App builder job: directory contains unsupported mime type"
+      value: "Unsupported MIME Type(s) detected in %{file_names}"
```

Comment:
nit: 
- sentence-case 't' in `type`
- trailing period at end of error sentence (optional, seems inconsistent)

otherwise 👍

Reply:
Ah actually you _could_ avoid writing `type(s)` and instead use the `.one` `.other` i18n magic here to take care of the pluralisation here if you wanted

Reply:
Okay let me give that a go